pkg/resources/etcd/statefulset.go

```diff
@@ -240,6 +247,11 @@ func StatefulSetCreator(data etcdStatefulSetCreatorData, enableDataCorruptionChe
 					},
 				},
 			}
+
+			set.Spec.Template.Spec.Containers[0].VolumeMounts = append(set.Spec.Template.Spec.Containers[0].VolumeMounts, resources.GetHostCACertVolumeMounts()...)
```

Comment:
is this dup of https://github.com/kubermatic/kubermatic/pull/5668/commits/41a43e33a3f8ffddbc4d2ab8fd0d8d43541fba3c#diff-5e4f08420bfa19f5d6cd42269a047618R340

Reply:
Hm no, one is the volumes, the other is the volume mounts.